qa/tasks/cephfs/test_misc.py

```diff
@@ -276,39 +276,39 @@ def _test_sync_stuck_for_around_5s(self, dir_path, file_sync=False):
                 self.mount_a.run_shell(["mkdir", os.path.join(dir_path, f"{i}_{j}")])
             start = time.time()
             if file_sync:
-                self.mount_a.run_shell(['python3', '-c', sync_dir_pyscript])
+                self.mount_a.run_shell(['python3', '-c', sync_dir_pyscript], timeout=4)
             else:
-                self.mount_a.run_shell(["sync"])
+                self.mount_a.run_shell(["sync"], timeout=4)
+            # the real duration should be less than the rough one
             duration = time.time() - start
-            log.info(f"sync mkdir i = {i}, duration = {duration}")
-            self.assertLess(duration, 4)
```

Comment:
We are removing this assert because now we are already providing a timeout of 4 to shell,now there's no need to check, right?

Reply:
Yeah, I just leave the debug logs to help us to understand what has happened.